### PR TITLE
[ADD] reconciled filter and new search_defaults

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

File: l10n_it_ricevute_bancarie/account/account_view.xml [hunk 1]
```diff
@@ -103,7 +106,7 @@
         <field name="view_type">form</field>
         <field name="view_mode">tree,form</field>
         <field name="view_id" ref="view_riba_da_emettere_tree" />
-        <field name="context">{'search_default_da_emettere':1, 'search_default_to_reconcile':1}</field>
+        <field name="context">{'search_default_da_emettere':1, 'search_default_reconciled':1, 'search_default_unsolved':1}</field>
```
Comment (eLBati) [inline l10n_it_ricevute_bancarie/account/account_view.xml]: Ripensandoci , questo è meglio non modificarlo: di default è meglio vedere quelle da emettere e da riconciliare

Comment (tafaRU) [inline l10n_it_ricevute_bancarie/account/account_view.xml]: @eLBati  , I've just pushed a new commit, plz have a look at it! Thanks
I also tested the new changes locally and they seem to work.

Comment (tafaRU) [inline l10n_it_ricevute_bancarie/account/account_view.xml]: @eLBati  plz have a look at my new commit. I tested it locally and it seems to work.

**Comment by tafaRU** [comment]
@eLBati plz have a look at my new commit. I tested it locally and it seems to work.

**Pull request merged**